### ISSUE TITLE
CI and Testing Github Integration

### ISSUE THREAD

**Issue by Squareys** [knime-ip/knip-scijava#50]
Personal wish of mine:

It would be amazing if we could have some kind of CI which builds branches and puts results onto the PRs. Usually, CIs have Github integrations which do this. Examples are AppVeyor or Travis, which are available for free for open source projects.

In addition, one could setup coveralls.io or something similar to provide feedback about test converage.

@dietzc I think we tried to find a simple way to get Github integrated with our existing Jenkins CI already, right? Since that doesn't seem easy to do, I propose to have a second CI (without artifact ouput) to just build and run tests for PRs etc.

I might try to get tests on Travis running after #49 is done. Since it pretty much allows to just run unix commands, it should be rather simple to do.

Opinions/wishes/suggestions are welcome.
Regards, Jonathan.

**Comment by dietzc** [comment]
yes, this would be nice. but please post-pone this issue until _all_ other issues have been resolved and we have something stable running.

**Comment by Squareys** [comment]
> have something stable running

It would definitely help with stability ;)

**Comment by dietzc** [comment]
... as soon as we have a first stable thing I agree we should do that. But please lets first make progress on the implementation front :+1:
